spec/facility_creator_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'spec_helper'
+
+RSpec.describe FacilityCreator do
+  describe 'create_facility' do
+    before do
+      @facilities_info = [
+        { address: '123 Main St', city: 'Anytown', state: 'ST', zip: '12345', phone: '555-1234' },
+        { address: '456 Elm St', city: 'Othertown', state: 'ST', zip: '54321', phone: '555-5678' }
+      ]
+    end
+
+    it 'creates a facility' do    
+      facility_creator = FacilityCreator.new
+
+      facilities = facility_creator.create_facilities(@facilities_info)
```

Comment:
I like that you're using fake data to test your method. However, because you are using fake data, that means you know exactly what data the Facility objects will have and we should be testing for that. So on line 19, in addition to testing that you've created a facility, we want to test that all data got set correctly so we can add `expect(facility.address).to eq("123 Main St"), etc. That way we know our method is properly assigning these values.